grails-app/controllers/org/pih/warehouse/inventory/StockMovementController.groovy

```diff
@@ -283,6 +286,87 @@ class StockMovementController {
         render(template: "documents", model: [stockMovement: stockMovement])
     }
 
+    def comments = {
+        Requisition requisition = Requisition.get(params.id)
+        render(template: "comments", model: [requisition: requisition])
+    }
+
+    def addComment = {
+        Requisition requisition = Requisition.get(params.id)
+        StockMovement stockMovement = StockMovement.createFromRequisition(requisition)
```

Comment:
I think this is not needed because show action will do it on its own again and I think addComment don't have to use `stockMovement` (correct me if I am wrong)

Reply:
Oh ok, this makes sense to me now.